build.psm1

```diff
@@ -587,6 +587,13 @@ Fix steps:
 
             try {
                 Push-Location $globalToolSrcFolder
+
+                if ($Options.Runtime -like 'fxdependent*') {
+                    if ($Arguments -contains '/property:UseAppHost=true') {
+                        $Arguments = @($Arguments | Where-Object { $_ -notlike '/property:UseAppHost=true' })
+                    }
+                }
```

Comment:
Why not just delete it from here:
https://github.com/PowerShell/PowerShell/blob/328188ea6717bcbd1eef7b486f47fdec19c3e095/build.psm1#L464-L466

Reply:
this is required for pwsh.exe, the current change removes the exe for GlobalTool.Shim.exe